newdle/core/app.py

```diff
@@ -86,7 +86,8 @@ def create_app(config_override=None, use_env_config=True):
     _configure_app(app, from_env=use_env_config)
     if config_override:
         app.config.update(config_override)
-    _configure_multipass(app)
+    if not app.config['SKIP_LOGIN']:
```

Comment:
SKIP_LOGIN was failing because `_configure_multipass` redirects were in effect, so had to add this to be able to use this option

Reply:
Moved this code to https://github.com/indico/newdle/pull/220